agences-regionales-sante/grand-est/2020-03-26.yaml

```diff
@@ -0,0 +1,12 @@
+date: 2020-03-26
```

Comment:
time tag is missing! Ils ont preciser 14H dans le pdf
Pour rappel on ajoute la balise time si le temps est renseingné cf [readme](https://github.com/opencovid19-fr/data)

Reply:
Merci ! C'est ajouté (force-push).